pkg/Directory.Build.props

```diff
@@ -7,6 +7,15 @@
     <IncludeSymbols>false</IncludeSymbols>
   </PropertyGroup>
 
+  <!-- nuspec properties -->
+  <PropertyGroup>
```

Comment:
>PropertyGroup [](start = 3, length = 13)

Just curious, does anything here in the nuspec make it clear to consumers of the nuget not only what repository it comes from, but what the commithash was on build? Not sure if that's buried somewhere in here, if it was it wasn't obvious.

Reply:
There is a new NuGet feature with the `2.1.200` SDK that just got [merged](https://github.com/dotnet/machinelearning/pull/105) that will put the commit hash and repo in the nuspec file.

I'll update this PR to set these properties as well.

See these random collection of links about this:
* https://github.com/NuGet/Home/issues/6547
* https://github.com/NuGet/NuGet.Client/pull/2081
* https://github.com/dotnet/designs/blob/master/accepted/diagnostics/source-link.md
* https://github.com/dotnet/designs/blob/master/accepted/build/standard-ci-env-variables.md

Reply:
Very nice, thanks @eerhardt .